docs/pages/management/dynamic-resources/terraform-provider.mdx

```diff
@@ -121,6 +126,7 @@ paste in the following content:
 
    tctl bots rm terraform-test
    rm -rf ./terraform-identity
+   rm -rf ./tbot-data
```

Comment:
```suggestion
   rm -rfv ./tbot-data
```

Nit: I try adding verbose to `rm -rf` in case I run it on a wrong thing... this way at least I can see what got removed.